x/stake/keeper/distribute_sidechain.go

```diff
@@ -196,7 +196,7 @@ func (k Keeper) DistributeInBreathBlock(ctx sdk.Context, sideChainId string) sdk
 			if k.AddrPool != nil {
 				k.AddrPool.AddAddrs(changedAddrs[:])
 			}
-			rewardSum += totalReward
+			rewardSum += remainReward.RawInt()
```

Comment:
Not get it. To fix a known issue?

Reply:
Yes. The recon service will compare between this with the sum of reward_details. And the reward_detail doesn't contain the validator's part. So I subtract that part from the total reward too.